introduction.md

```diff
@@ -0,0 +1,2 @@
+My name is Jean Carlos. I am passionate for software development and learn new things. 
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.